polyphemus/lib/etls/ipi/linkers/ipi_rna_seq_files_linker_base.rb

```diff
@@ -3,9 +3,8 @@
 
 class Polyphemus::IpiRnaSeqFilesLinkerBase < Polyphemus::MetisFilesLinkerBase
   def initialize(project_name: "ipi", bucket_name:, attribute_regex:, record_name_regex:)
-    super(project_name: project_name, bucket_name: bucket_name)
+    super(project_name: project_name, bucket_name: bucket_name, record_name_regex: record_name_regex)
```

Comment:
Making record_name_regex an attribute of the base linker really makes things simpler in the end, rather than this subclass.  See other places where it disappears since it is now guaranteed in the constructor.